src/components/sign/SignIn.js

```diff
@@ -0,0 +1,71 @@
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import axios from "axios";
+
+const SignIn = ({handleSign , isSigned}) => {
+
+    const formik = useFormik({
+        initialValues: {
+            email: "",
+            password: ""
+        },
+
+        validationSchema: Yup.object({
+            email: Yup.string()
+                .min(10,'invalid Email Adress Or Phone Number') 
+                .required('Required!!'),
+            password: Yup.string()
+                .required('Required!!')
+                .min(8, 'Password is too short - should be 8 chars minimum.')
+                .matches(/[a-zA-Z]/, 'Password can only contain Latin letters.')
+        }),
+
+        onSubmit: () => {
+            axios.post('http://localhost:5000/auth/signin', {
+                email: formik.values.email,
+                password: formik.values.password,
+            })
+                .then((res) => {
+                    handleSign(res.data);
+                    isSigned(false)
+                    console.log(res.data);
+
+                }).catch((e) => {
+                    console.log('server axios', e.response.data.message);
+                    console.log(e);
```

Comment:
this should be replaced later on with error component, formik has custom error component, you can check it from here [link](https://formik.org/docs/api/errormessage)